docs/guides/deploy-hyperlane-local-agents.mdx

```diff
@@ -0,0 +1,397 @@
+import TerminologyPartial from "/docs/partials/deploy-hyperlane/_terminology.mdx";
+import SetupKeysPartial from "/docs/partials/deploy-hyperlane/_setup-keys.mdx";
+import DeployContractsPartial from "/docs/partials/deploy-hyperlane/_deploy-contracts.mdx";
+import SendTestMessagesPartial from "/docs/partials/deploy-hyperlane/_send-test-messages.mdx";
+import DeployWarpRoutePartial from "/docs/partials/deploy-hyperlane/_deploy-warp-route.mdx";
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Deploy Hyperlane with Local Agents
+
+:::tip
+
+This guide is for advanced users who have some understanding of configuring and deploying Hyperlane.
```

Comment:
This might make people think that the local agents guide is the production guide, will try tweaking this idea though because I think it's along the right lines

Reply:
lmk what you think
```md
This guide is for advanced users who may eventually intend to run Hyperlane agents in a production-like environment. It will cover the basics of how to manually configure and run agents but **it is not a production setup guide**.
```

Reply:
sounds good, I don't feel too strongly but just wanted to highlight that this is very close to what it looks like to run in prod

Reply:
> that's kind of what the intention is, to use kurtosis for short-lived testing and run agent binaries manually for a longer-lived testnet / mainnet

I don't disagree, I'm just cautious about making sure people don't do key management like this in prod